debezium-jdbc-es/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#https://raw.githubusercontent.com/debezium/debezium-examples/master/unwrap-smt/debezium-jdbc-es/Dockerfile
+
+FROM debezium/connect:1.0
+ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc \
+    KAFKA_CONNECT_ES_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-elasticsearch
+
+ARG POSTGRES_VERSION=42.2.8
+ARG KAFKA_JDBC_VERSION=5.3.1
+
+# Deploy PostgreSQL JDBC Driver
+RUN cd /kafka/libs && curl -sO https://jdbc.postgresql.org/download/postgresql-$POSTGRES_VERSION.jar
```

Comment:
We might not need the sink connector including PostgresSQL JDBC driver and Kafka Connnect JDBC here

Reply:
Yes, I used sink for testing before but when our solution didn't use sink and I forgot to remove it before creating PR.
We can use  the pure debezium/connect:1.0 image